### DISCUSSION TITLE
Screen Mirror active - tasker integration?

### DISCUSSION THREAD

**Discussion by beer-matt** [BimmerGestalt/AAIdrive#837] [Ideas]
Is it possible for tasker to detect when screen mirroring is active (or if not - can something be added e.g. send an intent when mirroring started/ended)? This would allow action to be taken e.g. force landscape orientation, start app/ensure app is running etc.?

**Comment by hufman** [comment]
Screen Mirror itself doesn't send any broadcasts, and I'm not seeing any public Intents that indicate when screen recording starts, nor any existing Tasker functionality to detect it. You are welcome to add this functionality!
